src/ui/Colors.js

```diff
@@ -4,6 +4,7 @@ export default {
 	grey: '#999',
 	lightGrey: '#eee',
 	darkGrey: '#444',
+	darkerGrey: '#333',
```

Comment:
Because of the way sRGB gamma operates, they are significantly different. I tried #444, did not like it.